app/scripts/modules/core/src/projects/project.less

```diff
@@ -73,3 +73,26 @@
   flex: 1 1 auto;
   flex-direction: column;
 }
+
+.flex-container-h {
+  display: flex;
+  flex-flow: row nowrap;
+
+  &.baseline {
+    align-items: baseline;
+  }
+}
+
+.flex-container-v {
+  display: flex;
+  flex-flow: column nowrap;
+}
+
+
+.flex-grow {
+  flex: 1 1 auto;
+}
+
+.flex-nogrow {
+  flex: 0 1 auto;
+}
```

Comment:
I like these new classes but they probably should go somewhere more general that `project.less`